### PR TITLE
Fixes #1196, lamps don't drop in survival

### LINKED ISSUE #1196: TFCT: Lamps don't drop if broken in survival
Fixing #1162 broke survival uses.

**Meta Info**
 - TFC Version: build 140

### PR DESCRIPTION
**Pull request by Gaelmare**
Fixes #1196, lamps don't drop in survival
Also fixed issue with NBT interpretation on file load.

### CODE REVIEW

**Review by DisasterMoo — APPROVED** [review]
Just a minor cleanup, but you can leave that to me

**Comment by Gaelmare** [comment]
Curious what cleanup is needed?

**Comment by dries007** [comment]
RIP my test server lamps :(

**Comment by Gaelmare** [comment]
Ikr. Have a PR to fix it in. My apologies.

File: src/main/java/net/dries007/tfc/objects/blocks/metal/BlockMetalLamp.java [hunk 1]
```diff
@@ -347,8 +347,11 @@ public boolean hasTileEntity(IBlockState state)
         return true;
     }
 
+    //Lifted from BlockFlowerPot
 
-    //Lifted from BlockTorch
+    public void breakBlock(World worldIn, BlockPos pos, IBlockState state) {
+        super.breakBlock(worldIn, pos, state);
+    }
```
Comment (DisasterMoo) [inline src/main/java/net/dries007/tfc/objects/blocks/metal/BlockMetalLamp.java:354]: This one

**Pull request merged**